core-bundle/tests/Translation/DataCollectorTranslatorTest.php

```diff
@@ -76,4 +77,40 @@ public function testDoesNotCollectMessagesIfTheDomainIsNotAContaoDomain(): void
 
         $this->assertSame('translation', $translator->trans('translation_key', [], 'message_domain', 'en'));
     }
+
+    public function testServiceIsResetable(): void
+    {
+        $originalTranslator = $this->createMock(SymfonyDataCollectorTranslator::class);
+
+        $originalTranslator
+            ->expects($this->any())
```

Comment:
CS: remove `->expects($this->any())` as it is the default value.

Reply:
removed in 5ba4e1ec97fc5d592da43045402a34718b9ccb14